chapters/VK_EXT_debug_utils.txt

```diff
@@ -566,13 +575,13 @@ include::../api/structs/VkDebugUtilsMessengerCallbackDataEXT.txt[]
     conditions.
   * pname:queueLabelCount is a count of items contained in the
     pname:pQueueLabels array.
-  * pname:pQueueLabels is NULL or a pointer to an array of
+  * pname:pQueueLabels is `NULL` or a pointer to an array of
```

Comment:
I just saw in another section that this should be `code:NULL` (and all others as well).

Reply:
Styleguide says backticks.

Reply:
Really?  Wow, okay, never mind then.  Thanks for checking it out, though.